lib/galaxy/tool_util/deps/mulled/mulled_build.py

```diff
@@ -53,10 +54,7 @@
 DEFAULT_EXTENDED_BASE_IMAGE = os.environ.get(
     "DEFAULT_EXTENDED_BASE_IMAGE", "quay.io/bioconda/base-glibc-debian-bash:latest"
 )
-if "DEFAULT_MULLED_CONDA_CHANNELS" in os.environ:
-    DEFAULT_CHANNELS = os.environ["DEFAULT_MULLED_CONDA_CHANNELS"].split(",")
-else:
-    DEFAULT_CHANNELS = ["conda-forge", "bioconda"]
+DEFAULT_CHANNELS = default_mulled_conda_channels_from_env() or ["conda-forge", "bioconda"]
```

Comment:
Would it make sense to add this to the command line parser as well ?

Reply:
How do you mean this?

Reply:
There's an argparse section below, you'd want to be able to set the channels there